static/gsApp/views/amCheckout/index.tsx

```diff
@@ -427,6 +434,20 @@ class AMCheckout extends Component<Props, State> {
       onDemandBudget: parseOnDemandBudgetsFromSubscription(subscription),
     };
 
+    if (
+      isNewPayingCustomer(subscription, organization) &&
+      checkoutTier === PlanTier.AM3
+    ) {
+      // TODO(isabella): Test if this behavior works as expected on older tiers
```

Comment:
Are these `TODO`s intended to be merged or will you remove them after local testing?

Reply:
we intended the improvement for the latest tier of checkout but i was going to follow up after this PR to add the default PAYG behavior to older tiers too but just haven't tested those yet, hence the TODO to remind me 😁